presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/ResourceGroupsDao.java

```diff
@@ -81,18 +81,19 @@
             ")")
     void createSelectorsTable();
 
-    @SqlUpdate("CREATE TABLE IF NOT EXISTS exact_match_selectors(\n" +
+    @SqlUpdate("CREATE TABLE IF NOT EXISTS source_exact_match_selectors(\n" +
```

Comment:
How about `exact_match_source_selectors`? That way, if we have multiple tables, they will be grouped together.